ui/README.md

```diff
@@ -1,5 +1,5 @@
 # Job Manager UI
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 2.2.3.
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 1.6.5.
```

Comment:
Not sure if this change is correct, but since the latest release of angular-cli is v1.6.7, I guess 2.2.3 should be a typo?

Reply:
Yup good catch, I believe this was a mix up with the swagger-cli which is version 2.2.3